src/Core/Grid/Query/AliasQueryBuilder.php

```diff
@@ -87,7 +84,7 @@ public function getCountQueryBuilder(SearchCriteriaInterface $searchCriteria): Q
      *
      * @return QueryBuilder
      */
-    private function getAliasQueryBuilder(SearchCriteriaInterface $searchCriteria): QueryBuilder
+    public function getAliasQueryBuilder(SearchCriteriaInterface $searchCriteria): QueryBuilder
```

Comment:
This internal method should not be public just to help testing (I detailed a bit more in the behat code)